xtask/src/util.rs

```diff
@@ -301,7 +301,7 @@ pub fn gha_output(tag: &str, content: &str) {
         // https://github.com/actions/toolkit/issues/403
         panic!("output `{tag}` contains newlines, consider serializing with json and deserializing in gha with fromJSON()")
     }
-    gha_output!("::set-output name={tag}::{}", content);
+    gha_output!("tag={} >> $GITHUB_OUTPUT", content);
```

Comment:
this is not correct, `gha_output` is simply an alias to println/eprintln

Instead we need to write to the file pointed to from `GITHUB_OUTPUT`